src/Dropdown.tsx

```diff
@@ -90,10 +90,24 @@ export interface DropdownProps {
   alignEnd?: boolean;
   defaultShow?: boolean;
   show?: boolean;
-  onToggle: (nextShow: boolean, event?: React.SyntheticEvent) => void;
+  onToggle: (nextShow: boolean, event?: React.SyntheticEvent | Event) => void;
   itemSelector?: string;
   focusFirstItemOnShow?: false | true | 'keyboard';
-  children: (arg: { props: DropdownInjectedProps }) => React.ReactNode;
+  children: React.ReactNode;
+}
+
+function useRefWithUpdate() {
+  const forceUpdate = useForceUpdate();
```

Comment:
**question:** do these get properly enqueued/deduped, or could we end up with a double-rerender?

Reply:
I'm fairly sure it does, maybe not a big deal either way idk